lib/rules/no-unknown-property.js

```diff
@@ -69,33 +69,10 @@ const ATTRIBUTE_TAGS_MAP = {
   align: ['applet', 'caption', 'col', 'colgroup', 'hr', 'iframe', 'img', 'table', 'tbody', 'td', 'tfoot', 'th', 'thead', 'tr'], // deprecated, but known
   valign: ['tr', 'td', 'th', 'thead', 'tbody', 'tfoot', 'colgroup', 'col'], // deprecated, but known
   noModule: ['script'],
-  // Media events allowed only on audio and video tags, see https://github.com/facebook/react/blob/256aefbea1449869620fb26f6ec695536ab453f5/CHANGELOG.md#notable-enhancements
-  onAbort: ['audio', 'video'],
-  onCancel: ['dialog'],
-  onCanPlay: ['audio', 'video'],
-  onCanPlayThrough: ['audio', 'video'],
-  onClose: ['dialog'],
-  onDurationChange: ['audio', 'video'],
-  onEmptied: ['audio', 'video'],
+  // Note: Most events can be added to any HTML element, even if not specified in the standard
+  // see https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes
```

Comment:
i think our standard should be higher than "whatever wacky things HTML allows" in a linter whose goal is to prevent bugs.

Reply:
Yeah that's fair. I can update it to follow a more sensible philosophy. I do agree that the HTML spec is overly permissive with these.